tests/e2e/testfiles/scenarios.yaml

```diff
@@ -1802,7 +1802,6 @@
     - "Set up reverse proxy tls secret namespace [powermax]"
     - "Set up creds with template [testfiles/powermax-templates/powermax-secret-template.yaml] for [pmaxCreds]"
     - "Set up configMap with template [testfiles/powermax-templates/powermax_reverse_proxy_config.yaml] name [powermax-reverseproxy-config] in namespace [powermax] for [pmaxAuthSidecar]"
-    - "Set up creds with template [testfiles/powermax-templates/powermax-array-config.yaml] for [pmaxArrayConfig]"
```

Comment:
instead of removing these lines, they should be adjusted to work with the pmax CR being tested. 
So in this case, this line should be changed to:
```
- "Set up creds with template [testfiles/storage_csm_powermax.yaml] for [pmaxArrayConfig]"
```

Reply:
Taken care here: https://github.com/dell/csm-operator/pull/910